v0/destinations/pardot/data/PardotIdentify.json

```diff
@@ -0,0 +1,277 @@
+[
+    {
+        "destKey": "email",
+        "sourceKeys": [
+      "traits.email",
+      "context.traits.email"
+        ],
+        "required": true
+      },
+      {
+        "destKey": "campaign_id",
+        "sourceKeys": [
+      "campaignId",
+      "traits.campaignId",
+      "traits.campaign_id",
+      "context.traits.campaignId",
+      "context.traits.campaign_id"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "salutation",
+        "sourceKeys": [
+      "traits.salutation",
+      "context.traits.salutation"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "first_name",
+        "sourceKeys": [
+      "traits.first_name",
+      "context.traits.first_name",
+      "traits.firstName",
+      "context.traits.firstName"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "last_name",
+        "sourceKeys": [
+      "traits.last_name",
+      "context.traits.last_name",
+      "traits.lastName",
+      "context.traits.lastName"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "password",
```

Comment:
Is it a required field? Ideally, no one sends `password` in the `traits`

Reply:
removed